config.go

```diff
@@ -941,6 +946,14 @@ func loadConfig() (*config, []string, error) {
 		}
 	}
 
+	// Warn if old testnet directory is present.
+	for _, oldDir := range oldTestNets {
+		if fileExists(oldDir) {
+			dcrdLog.Warnf("Data from previous version of testnet found"+
+				" at: %v", oldDir)
```

Comment:
How about:

Block chain data from previous testnet found (/path/to/testnet) and can probably be removed.

?